collector/lib/ProcfsScraper.cpp

```diff
@@ -407,7 +413,7 @@ bool ReadContainerConnections(const char* proc_path, ProcessStore* process_store
   // Read all the information from proc.
   while (auto curr = procdir.read()) {
     if (!std::isdigit(curr->d_name[0])) continue;  // only look for <pid> entries
-    int pid = stoi(curr->d_name);
+    int pid = std::strtoll(curr->d_name, 0, 10);
```

Comment:
You are using strtoll but only using an int to store it, since the original code used atoi I would recommend using strtol instead, if the bigger integers are indeed needed change int for auto or long long.

Reply:
Changed pid to `long long`